specification/network/resource-manager/Microsoft.Network/stable/2021-02-01/virtualNetwork.json

```diff
@@ -1038,6 +1038,13 @@
             },
             "description": "Parameters supplied to the create or update virtual network peering operation."
           },
+          {
+            "name": "syncRemoteAddressSpace",
```

Comment:
I found this parameter is not used in examples. Can you add usage of it?

Reply:
Added it now. It's a query parameter. Thanks for catching this.

Reply:
The way we handle the query param should work in either way. We are not using quotes in the query params too. I have pinged you separately with more details. Please let me know if I need to provide the values in example to be a string, like "true".

Reply:
> The way we handle the query param should work in either way. We are not using quotes in the query params too. I have pinged you separately with more details. Please let me know if I need to provide the values in example to be a string, like "true".

Swagger ModelValidation is failing because of that. I think you should use string here.

Reply:
BTW, this property should be enum, as its value could only be "true" or "false". Like following

https://github.com/Azure/azure-rest-api-specs/blob/e4c056151df3c59bbb2221e23f623fdb625313c5/specification/apimanagement/resource-manager/Microsoft.ApiManagement/stable/2017-03-01/apimapis.json#L3489-L3502